### PR TITLE
DOC-597: Create query worker source and sink docs

### PR DESCRIPTION
**Pull request by diana-macrometa**
- Creates query worker source and sink
- Moved stream-sink up one level
- Added and updated links.

@StoyanTc - a lot of content is me inferring based on examples. Please read through the new topics very thoroughly!

### CODE REVIEW

File: docs/cep/sink/query-worker-sink.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+```sql
+CREATE SINK <NAME> WITH (type="query-worker", map.type="<STRING>", query.worker.name="<STRING>", sink.id="<STRING>")
+```
+
+## Query Parameters
+
+| Name     | Description                   | Default Value            | Possible Data Types | Optional | Dynamic |
+|----------|---------------------------|--------------------------|---------------------|----------|---------|
+| query.worker.name  | TThe name of an existing query worker. | | string | No | Yes |
```
Comment (jamesm-macrometa) [inline docs/cep/sink/query-worker-sink.md:20]:
```suggestion
| query.worker.name  | The name of an existing query worker. | | string | No | Yes |
```

Comment (jamesm-macrometa) [reply docs/cep/sink/query-worker-sink.md:20]: Is one column supposed to be empty here?

**Review by jamesm-macrometa — APPROVED** [review]
Looks great!  Just a couple minor suggestions.

File: docs/cep/sink/query-worker-sink.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/cep/sink/query-worker-sink.md:20]: Yes, because there is no default value for either.

**Comment by diana-macrometa** [comment]
Heya @StoyanTc! Will you be able to review this one soon?

**Review by StoyanTc — APPROVED** [review]

**Pull request merged**